pkg/server/middleware/tracer.go

```diff
@@ -43,29 +44,29 @@ func (t Tracer) Wrap(next http.Handler) http.Handler {
 
 // ExtractTraceID extracts the trace id, if any from the context.
 func ExtractTraceID(ctx context.Context) (string, bool) {
-	sp := opentracing.SpanFromContext(ctx)
-	if sp == nil {
-		return "", false
-	}
-	sctx, ok := sp.Context().(jaeger.SpanContext)
-	if !ok {
-		return "", false
-	}
-
-	return sctx.TraceID().String(), true
+	traceID, _ := ExtractSampledTraceID(ctx)
+	return traceID, true
```

Comment:
the logic of `ExtractTraceID` has changed. Now it always returns true in the second parameter. But it probably shouldn't when ExtractSampledTraceID returns `"", false`

Reply:
(That was actually part of the original code - I'll remove the method entirely, it just returned "true" as the "ok" return value (not sampled)).

Reply:
No I won't, in fact... It's actually documented as doing that...
```
// ExtractSampledTraceID works like ExtractTraceID but the returned bool is only
// true if the returned trace id is sampled.
```
=> the return value for the other method is "true" (always)
Ignore my previous comment